src/bitExpert/PHPStan/Magento/Autoload/ProxyAutoloader.php

```diff
@@ -110,7 +110,9 @@ protected function getFileContents(string $class): string
                                 $defaultValue = ' = false';
                                 break;
                             default:
-                                $defaultValue = ' = ' . $parameter->getDefaultValue();
+                                if (is_string($parameter->getDefaultValue())) {
+                                    $defaultValue = ' = ' . $parameter->getDefaultValue();
+                                }
```

Comment:
Not sure if this is a good solution, `$parameter->getDefaultValue()` returns `mixed`, so phpstan complained about this concatenation not being done on a string.